style.css

```diff
@@ -0,0 +1,247 @@
+  line-height: 20px;
+  color: #6070ff;
+  padding-top: 15px;
+  padding-left: 30px;
+}
+
+.hum-menu {
+  padding-top: 38px;
+  padding-right: 20px;
+}
+
+.headerline {
+  background: url(./assets/img/header-shapes\ mobile.png);
+  height: 90%;
+  display: flex;
+  flex-direction: column;
+  background-repeat: no-repeat;
+  border-bottom-left-radius: 8px;
+}
+
+.headerline-detail {
+  margin: 80px 12px 0 12px;
+  width: 327px;
+}
+
+.name {
+  margin: 0;
+  font-weight: 800;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.aboutme {
+  margin: 0;
+  font-weight: 400;
+  color: #344563;
+}
+
+.secondary-test .supporting-test {
+  line-height: 24px;
+}
+
+.connect {
+  font-weight: 500;
+  color: #7f8cff;
+  text-transform: uppercase;
+}
+
+.social-media-icons {
+  list-style-type: none;
+  padding-left: 0;
+  width: 170px;
+  height: 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+/* in mobile first approch this have no effect */
+.hum-menu,
+li:hover {
+  cursor: pointer;
+}
+
+.works {
+  background-color: #e5e5e5;
+  width: 100%;
+  padding: 150px 24px 150px 24px;
+  display: grid;
+  grid-gap: 88px;
+}
+
+.card-works {
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 15px;
+}
+
+.card-details {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+.project-title {
+  margin: 0;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.frame-item {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.frame-item > :first-child {
+  list-style-type: none;
+  text-transform: uppercase;
+  color: #7a869a;
+}
+
+.frame-item li:nth-child(2),
+.frame-item li:nth-child(3),
+.frame-item li:nth-child(4) {
+  margin-left: 25px;
+  color: #344563;
+}
+
+.p-detail {
+  margin: 0;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  color: #344563;
+}
+
+.cats {
+  margin: 0;
+  list-style-type: none;
+  width: 200px;
+  height: 24px;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+}
+
+.cats li:nth-child(1),
+.cats li:nth-child(2),
+.cats li:nth-child(3) {
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  font-size: 12;
+  line-height: 16px;
+  display: flex;
+  align-items: center;
+  color: #6070ff;
+}
+
+.btn {
+  width: 135px;
+  height: 35px;
+  color: #6070ff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-size: 17px;
+  background: transparent;
+}
+
+.btn:hover {
+  cursor: pointer;
+  background-color: #6070ff;
+  color: #fff;
+  transition: all 0.6s;
+}
+
+.btn:disabled {
+  cursor: none;
+}
+
+.about {
+  margin-top: 30px;
+  padding: 0 24px;
+}
+
+.language-list-item {
+  display: flex;
+  gap: 10px;
+  background: #f7f7f9;
+  padding: 10px;
+  margin-bottom: 10px;
+  border-radius: 8px;
+}
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided. Please add a margin between the get resume button and the languages section. :pray: :pray: 
Your design
![image](https://user-images.githubusercontent.com/98363075/198585415-41f70a4f-b3ef-4ff0-b9e5-b0fac17cde54.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/198585508-3ab6529d-f42d-46b6-88e4-e207a6d50a5a.png)